Formula/httpd.rb

```diff
@@ -18,14 +18,188 @@ class Httpd < Formula
   depends_on "pcre"
 
   def install
-    # use Slackware-FHS layout as it's closest to what we want.
-    # these values cannot be passed directly to configure, unfortunately.
-    inreplace "config.layout" do |s|
-      s.gsub! "${datadir}/cgi-bin", "#{pkgshare}/cgi-bin"
-      s.gsub! "${datadir}/error",   "#{pkgshare}/error"
-      s.gsub! "${datadir}/htdocs",  "#{pkgshare}/htdocs"
-      s.gsub! "${datadir}/icons",   "#{pkgshare}/icons"
-    end
```

Comment:
Change this to:

```
    inreplace "config.layout" do |s|
      s.gsub! "${datadir}/htdocs",  "${datadir}"
      s.gsub! "${datadir}/error",   "#{pkgshare}/error"
      s.gsub! "${datadir}/icons",   "#{pkgshare}/icons"
    end
```

I want to have us use the same `var/www` root default as `lighttpd` and `nginx` use and only put regularly user-modifiable files and directories there.

Reply:
That's fine, I'll make sure the opt paths get put in the config files for the share directory.